### ISSUE TITLE
[KEYBOARD] 508-defect-1: VAOS appointment tabs MUST be keyboard accessible

### ISSUE THREAD

**Issue by 1Copenut** [department-of-veterans-affairs/va.gov-team#15015]
# [508-defect-1 :exclamation: Launchblocker](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/defect-severity-rubric.md#508-defect-1)

**Feedback framework**

- **❗️ Must** for if the feedback must be applied 
- **⚠️ Should** if the feedback is best practice 
- **✔️ Consider** for suggestions/enhancements 

## Point of Contact

**VFS Point of Contact:** _Trevor_

## User Story or Problem Statement

As a keyboard user, I want to be able to press `TAB` and have the appointment tablist receive keyboard focus so I can review my upcoming, past, and Express Care appointments.

## Details

The appointments tablist links all have a `tabindex="-1"` on first page load. This means the first (Upcoming) tab cannot receive keyboard focus when users press `TAB`, and makes the Past and Express Care tabs unavailable to keyboard users and other keyboard-like devices. Animated GIF attached below.

## Acceptance Criteria

- [ ] Current tab link does not have a `tabindex` attribute, or a `tabindex="0"` attribute
- [ ] Current tab will receive keyboard focus. This can be verified by a console `document.activeElement` and/or the yellow focus halo.
- [ ] Arrow keys continue to behave as they do currently (no regression in behavior)

## Definition of done

1. Review and acknowledge feedback.
1. Fix and/or document decisions made.
1. Accessibility specialist will close ticket after reviewing documented decisions / validating fix.

## Environment

* https://staging.va.gov/health-care/schedule-view-va-appointments/appointments/

## Solution (if known)

```diff
<a
  aria-current="page"
  class="vaos-appts__tab vads-u-text-align--center vads-u-color--gray-dark vaos-appts__tab--current"
  id="tabupcoming"
  aria-selected="false"
  role="tab"
- tabindex="-1"
  href="/health-care/schedule-view-va-appointments/appointments/"
>Upcoming</a>
```

## WCAG or Vendor Guidance (optional)

* [Keyboard: Understanding SC 2.1.1](https://www.w3.org/TR/UNDERSTANDING-WCAG20/keyboard-operation-keyboard-operable.html)

## Screenshots or Trace Logs

![tabindex-error-appointments](https://user-images.githubusercontent.com/934879/96520605-0fa57380-1235-11eb-89a5-5692c4d2605f.gif)

**Comment by jbalboni** [comment]
@1Copenut This should be on staging this afternoon for retesting, if that's necessary.

**Comment by 1Copenut** [comment]
@jbalboni I'm confident in the fix. I tested it when your PR branch was opened, so we should be 💯
